cfgov/unprocessed/js/organisms/MegaMenuMobile.js

```diff
@@ -171,7 +224,14 @@ function MegaMenuMobile( menus ) {
    */
   function resume() {
     if ( _suspended ) {
-      treeTraversal.bfs( _menus.getRoot(), _setTransition );
+      var transition = new MoveTransition( _rootMenu.getDom().content ).init();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1617/commits/ba618d387db8265534298e94c02f606c05b0def5